gqlex.g4

```diff
@@ -40,19 +76,19 @@ pathExpression
     ;
 
 rootPathExpression
-    : (ALL | NODE) (pathElement (NODE pathElement)*)+
+    : (DOUBLE_SLASH | SLASH) (pathElement (SLASH pathElement)*)?
```

Comment:
@yaronTTkarni Do we allow `//` (no range, no path elements) as a valid expression?

Reply:
yes, most simple query